src/Context.sol

```diff
@@ -37,6 +37,36 @@ enum CallType {
     UNKNOWN
 }
 
+/// @notice Allows ordering of the blockchain events
+struct OrdinalComponents {
```

Comment:
The comments are mostly rewording from the docs we had with sim studio https://docs.sim.io/docs/global-counter#what-it-is

The fields we extract are the same that Tal suggested in the linear ticket.